src/lib/hooks/data/useUtbetalinger.tsx

```diff
@@ -10,17 +10,17 @@ export const useUtbetalinger = () => {
 
     const setBekreftelse = async (bekreftelse: boolean) => {
         if (!utbetalinger) return;
-        const oppdatert = {...utbetalinger, bekreftelse};
```

Comment:
Småpirk, men: Hvorfor denne? De to versjonene er identiske.

Reply:
Å, siden avhukingen til checkbox som var under "Andre inntekter" forsvant etter en navigering, og siden checkbox avhukingen under "Annen formue" ikke forsvant etter en navigering brukte jeg useVerdier.tsx som eksempel til å løse problemet. Så det ble mer eller mindre copy-past for å være helt sikker på at problemet ble løst :)